frontend/src/pages/home/Homepage.tsx

```diff
@@ -0,0 +1,54 @@
+import './homepage.css'
+import LiveMeeting from './components/live_meeting/LiveMeeting';
+import UpcomingMeeting from './components/UpcomingMeeting';
+import PostMeeting from './components/PostMeeting';
+import { useMeetingContext } from '../../contexts/MeetingContext';
+
+export default function Homepage() {
+
+  const meetingContext = useMeetingContext();
+
+  const renderLiveMeetings = () => {
+    const now = Date.now();
+    return Object.values(meetingContext.meetings)
+      .filter(meeting => meeting.time <= now && meeting.time + 3_600_000 >= now)
+      .map(meeting => <LiveMeeting key={meeting.id} meeting={meeting} />)
+  }
+
+  const renderUpcomingMeetings = () => {
+    const now = Date.now();
+    return Object.values(meetingContext.meetings)
+      .filter(meeting => meeting.time >= now)
+      .map(meeting => <UpcomingMeeting key={meeting.id} meeting={meeting} />)
+  }
+
+  const renderPostMeetings = () => {
+    const now = Date.now();
+    return Object.values(meetingContext.meetings)
+      .filter(meeting => meeting.time + 3_600_000 <= now)
```

Comment:
Maybe move `const now = Date.now()` out to where the `meetingContext` is so that it can be reused by all of these functions.